src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.setAttribute('class', `notification ${type}`);
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const titleElement = document.createElement('h1');
```

Comment:
The `titleElement` should be an `h2` element instead of an `h1` element, as specified in the task requirements. Please change `document.createElement('h1')` to `document.createElement('h2')`.